src/org/usfirst/frc/team4571/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,58 @@
+package org.usfirst.frc.team4571.robot.subsystems;
+
+import org.usfirst.frc.team4571.robot.RobotConstants;
+import com.ctre.CANTalon;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
+ */
+public class IntakeSubsystem extends Subsystem {
+	private CANTalon intakeMotor;
+	private boolean isRollerIn;
+	private boolean isRollerOut;
```

Comment:
We dont need both booleans, because isRollerIn = false means that roller is out and isRollerIn = true means that the roller is in. Lets remove one of them.